media-gfx/img2pdf/files/img2pdf-0.4.0_disable-error-tests.patch

```diff
@@ -0,0 +1,33 @@
+diff -Naru a/src/img2pdf_test.py b/src/img2pdf_test.py
```

Comment:
Source for the patch / some bug references?

Reply:
In the ebuild I added reference
https://gitlab.mister-muffin.de/josch/img2pdf/issues/85

Reply:
We like to keep the headers **in** the patch file itself, because other distros/users may just look at the patch and not ebuild. It should tell everything relevant.